packages/fxa-admin-panel/package.json

```diff
@@ -38,7 +38,7 @@
     "convict": "^6.2.3",
     "convict-format-with-moment": "^6.2.0",
     "convict-format-with-validator": "^6.2.0",
-    "dateformat": "^4.3.1",
+    "dateformat": "^5.0.3",
```

Comment:
_Possibly_ relevant, but do you want to bump `@types/dateformat: "^3.0.1"` to `"^5.0.0"` while we're at it (L64 below).

```sh
npm info @types/dateformat version # 5.0.0
```